### PR TITLE
Add "Building ChatGPT with GPT-3" to applied prompting section

### PR DESCRIPTION
**Pull request by jayo78**

### CODE REVIEW

File: docs/applied_prompting/build_chatgpt.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+In the chat bot prompt above, I instruct Skippy to provide responses to users. GPT-3 picks up on the back-and-forth format, "USER: {user input}", SKIPPY: {skippy response}. Now when we provide the next input for a user, GPT-3 understands that Skippy is a chat bot and the previous exchanges are **a conversation.**
+
+### Memorization
+
+Past exchanges between Skippy and User get appended to  the next prompt. Each time we give more user input and get more chatbot output, the prompt expands to incorporate this new exchange. This is how chatbots like Skippy and ChatGPT can **remember what you previously asked it.**
+
+Prompts can get massive after several exchanges, especially if we are using the chat bot to generate long responses like blog posts. Prompts sent to GPT-3 are converted into tokens, which are numerical representations of words or parts of words. This allows GPT-3 to efficiently process large amounts of text. 
+
+There is a limit of 4097 tokens (about 3000 words) for the combined prompt and generated response for GPT-3 models, including ChatGPT. There are possible work arounds like using tactics like summarizing user inputs.
```
Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:52]: source for 4097?

Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:52]: `like using tactics like` rephrase

File: docs/applied_prompting/build_chatgpt.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+There are many different use cases of this prompting technique. ChatGPT is **optimized for answering questions efficiently with little context.** It will only ask follow ups if absolutely necessary.
```
Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:54]: `optimized for answering questions efficiently with little context` source?

File: docs/applied_prompting/build_chatgpt.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+<div style={{textAlign: 'left'}}>
+  <img src={Skippy} style={{width: "700px"}} />
+  <p style={{color: "gray", fontSize: "12px", fontStyle: "italic"}}>Full Skippy chat bot prompt from article header</p>
+</div>
+
+<a href="https://learnprompting.org/docs/basics/prompting">Prompting is the process of instructing an AI to do something. </a> As you have probably seen in ChatGPT examples online, you can prompt GPT-3 with just about anything. Common use cases are summarizing text, writing content based on a description, or creating things like poems, recipes, and much more.
+
+<p></p>
+
+**The core abstraction to this process is input/output.** You input information and then the model uses the data it was trained on to predict the best completion to your input. An important thing to note is that an LLM like GPT-3 isn't responding to a prompt, it is determining what tokens come next based on a probability distribution of tokens is has seen during pre-training(@jurafsky2009). Put simply, it is figuring out what words comes next given some previous words.
```
Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:47]: What does `The core abstraction to this process is input/output.` mean? Do you need this sentence?

File: docs/applied_prompting/build_chatgpt.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
```
Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:47]: This is a cool paragraph, but it is very technical compared to others. How does it help the reader understand how to build ChatGPT from GPT-3?

File: docs/applied_prompting/build_chatgpt.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+<p></p>
+
+**The core abstraction to this process is input/output.** You input information and then the model uses the data it was trained on to predict the best completion to your input. An important thing to note is that an LLM like GPT-3 isn't responding to a prompt, it is determining what tokens come next based on a probability distribution of tokens is has seen during pre-training(@jurafsky2009). Put simply, it is figuring out what words comes next given some previous words. 
+
+<p></p>
+
+GPT-3 is able to learn in context based on a few examples or a simple instruction. This is called few-shot learning (@brown2020language). In the chat bot prompt above, I instruct Skippy to provide responses to users. GPT-3 picks up on the back-and-forth format, `USER: {user input}` and `SKIPPY: {skippy response}`. GPT-3 understands that Skippy is a chat bot and the previous exchanges are a conversation so that when we provide the next user input, "Skippy" will respond.
```
Comment (trigaten) [inline docs/applied_prompting/build_chatgpt.md:51]: I really like how you noted the importance of few shot here. Very cool.

File: docs/applied_prompting/build_chatgpt.md (hunk 4 above)
Comment (jayo78) [reply docs/applied_prompting/build_chatgpt.md:47]: Supposed to lead into next pg about in context learning. Used to explain llms predict words and don't just respond to prompt. Chat bots are learned "behavior", gpt predicts based on few shot.

**Pull request merged**